draft-ietf-acme-acme.md

```diff
@@ -2455,7 +2455,7 @@ document ]]
 ### Identifier Types {#iana-identifier}
 
 This registry lists the types of identifiers that ACME clients may request
-authorization to issue in certificates.
+authorization to issue certificates.
```

Comment:
This registry lists the types of identifiers in certificates that ACME clients may request authorization to issue.